src/schedule/ucc_schedule.c

```diff
@@ -24,10 +24,10 @@ void ucc_event_manager_subscribe(ucc_event_manager_t *em, ucc_event_t event,
 
 ucc_status_t ucc_coll_task_init(ucc_coll_task_t *task)
 {
-    task->super.status   = UCC_OPERATION_INITIALIZED;
-    task->ee             = NULL;
-    task->was_progressed = 0;
-    task->flags          = 0;
+    task->super.status       = UCC_OPERATION_INITIALIZED;
+    task->ee                 = NULL;
+    task->lf_elem.was_queued = 0;
```

Comment:
@lappazos i think we should add a generic (maybe inline) function ucc_lf_queue_elem_init instead of directly initializing was_queued. LF queueue currently is a data structure that provides all the interfaces and hides the implementation. UCC schedule should not access internals like "was_queued" imho.